docs/directives.md

```diff
@@ -1,12 +1,42 @@
 # AngularJS directives in uportal-app-framework
 
+## Frame page
+
+Frame page should be used for all pages in a uportal-app-framework application. It comes with much of the rest of the framework's skeleton built in, including the app-header and main-menu directives. You should use it as the outermost element for
```

Comment:
:art: `app-header` and `main-menu` directives should have some form of code highlight, it looks like `**` is being used elsewhere.